daliuge-common/dlg/restutils.py

```diff
@@ -139,7 +139,7 @@ def __exit__(self, typ, value, traceback):
 
     def _get_json(self, url):
         ret = self._GET(url)
-        return json.load(ret) if ret else None
+        return json.load(ret) if ret else {}
```

Comment:
I have updated this to return a dictionary as we were not checking for `None` when returning the value in `get_json`. This means we could receive a `TypeError: NoneType is not iterable` error at runtime. 
Returning the dictionary iterable here is more 'Pythonic' as we are able to use the empty dictionary as an iterable regardless of how many elements are in it (i.e. duck-typing).